completions/man

```diff
@@ -46,9 +46,7 @@ _comp_cmd_man()
     [[ $was_split ]] && return
 
     if [[ $cur == -* ]]; then
-        COMPREPLY=($(
-            compgen -W '$(_parse_help "$1" -h || _parse_usage "$1")' -- "$cur"
-        ))
+        _comp_compgen_help "$1" -h || _comp_compgen_usage "$1"
```

Comment:
Maybe
```suggestion
        _comp_compgen_help -- -h || _comp_compgen_usage
```

Reply:
Applied in cb84b1977759d5c7bb32282b67765a0fc28c05d8 and squashed into 6677a21d50c93cafaaabc657cdd53ae33c149489